07.ls_object/ls.rb

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative 'lib/ls_command'
+
+LsCommand.new.show_ls_files
```

Comment:
```suggestion
LsCommand.new.show_files
```

LsCommandのメソッドなのでlsは省略してもいいかなーと思います。